pyia/data.py

```diff
@@ -427,8 +432,18 @@ def get_skycoord(self, distance=None, radial_velocity=None):
 
         kw = dict()
         if self._has_rv:
-            kw['radial_velocity'] = self.radial_velocity
-
+            # Fill nans with zeros
+            fill = ~np.isfinite(self.radial_velocity)
+            rv = self.radial_velocity*1
+            rv[fill] = 0*u.km/u.s
+            
+            kw['radial_velocity'] = rv
+            
+        # Reference epoch (2015.5 for DR2, could change in future)
+        obstime = astropy.time.Time(self.ref_epoch.value,
```

Comment:
I like the idea of having this! But I sometimes download data without this column. Maybe we can add a `default_ref_epoch` argument to the initializer that looks something like:
```python
def __init__(self, data, ref_epoch='latest', **kwargs):
    if ref_epoch == 'latest':
        # DR2, for now
        ref_epoch = Time(2015.5, format='decimalyear')
```